### DISCUSSION TITLE
Is it possible to hide specific directory?

### DISCUSSION THREAD

**Discussion by VisionaryAppDev** [nvim-tree/nvim-tree.lua#2026] [Q&A]
I have read though the docs but could find if it's possible to hide specific directory such as node_module or .git because when I need to expand directory, it took too long. I saw there is a shortcut to hide hidden file (`api.tree.toggle_hidden_filter`) but it doesn't serve my purpose since some file like .env should be shown.

**Comment by gegoune** [comment]
Is that within git repository? If so, are you not hit ignoring it?

**Reply by VisionaryAppDev** [reply]
Yes, the project contain .git directory.  Please see the below structure: 
```
my-express-project/
├── .git/
├── node_modules/
├── public/
│   ├── css/
│   ├── js/
│   └── images/
├── views/
│   ├── layouts/
│   └── partials/
├── routes/
│   ├── index.js
│   └── users.js
├── app.js
├── package.json
└── .env
```

If i were to hide hidden file such as .git, then it also hide .env file while node_module still visible there. So, when press Expanse All, it will take quite a lot of time and messy because node_module contains a lot of packages.

**Reply by gegoune** [reply]
You can set nvim-tree not to show gotignored nodes. Have a look at docs.
